lib/core/controlplane_api_direct.rb

```diff
@@ -65,47 +63,14 @@ def api_host(host)
     end
   end
 
-  # rubocop:disable Style/ClassVars
-  def api_token # rubocop:disable Metrics/MethodLength
-    return @@api_token if defined?(@@api_token)
-
-    @@api_token = {
-      token: ENV.fetch("CPLN_TOKEN", nil),
-      comes_from_profile: false
-    }
-    if @@api_token[:token].nil?
-      @@api_token = {
-        token: `cpln profile token`.chomp,
-        comes_from_profile: true
-      }
-    end
-    return @@api_token if @@api_token[:token].match?(API_TOKEN_REGEX)
+  def api_token
+    token = ENV.fetch("CPLN_TOKEN", nil) || `cpln profile token $CPLN_PROFILE`.chomp
+    return token if token.match?(API_TOKEN_REGEX)
```

Comment:
Instead of the previous complex way of refreshing the token, we can just fetch the token on every request. This will automatically refresh it if needed.

Reply:
This complex refresh was because it takes 0.5-2s !!! to run `cpln profile token` **every time on every api request** (and normally there are several).

Just delete your local `CPLN_TOKEN` and see how noticeably slower complex commands have become.

So.... I would keep it w/o changes here. CC @justin808